File: Components/WinForms/CS/SpreadsheetMainDemo/Modules/AutoFilter.cs
using System;
using DevExpress.XtraSpreadsheet.Demos;
using DevExpress.XtraBars.Ribbon;

namespace DevExpress.XtraSpreadsheet.Demos {
    public partial class AutoFilterModule : SpreadsheetRibbonTutorialControlBase {
        public AutoFilterModule() {
            InitializeComponent();
            spreadsheetControl1.Options.Culture = DefaultCulture;
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("AutoFilter_template.xlsx"));
        }

        #region Properties
        protected internal override RibbonPage SelectedRibbonPage { get { return dataRibbonPage1; } }
        #endregion
    }
}

File: Components/WinForms/VB/SpreadsheetMainDemo/Modules/AutoFilter.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraSpreadsheet.Demos
Imports DevExpress.XtraBars.Ribbon

Namespace DevExpress.XtraSpreadsheet.Demos
	Partial Public Class AutoFilterModule
		Inherits SpreadsheetRibbonTutorialControlBase
		Public Sub New()
			InitializeComponent()
			spreadsheetControl1.Options.Culture = DefaultCulture
			spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("AutoFilter_template.xlsx"))
		End Sub

		#Region "Properties"
		Protected Friend Overrides ReadOnly Property SelectedRibbonPage() As RibbonPage
			Get
				Return dataRibbonPage1
			End Get
		End Property
		#End Region
	End Class
End Namespace
